chaos_wrapper.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+function rollback() {
+    # avoid commits from the same faulty PR
+    rb_commit=$(git log --author chaosbot --format=format:%H | tail -n +2 | head -n 1)
```

Comment:
It looks like commits with author `chaosbot` happen when @amoffat manually fixes problems, his should be changed to `Chaos`